photosynthesis_metrics/ssim.py

```diff
@@ -159,7 +114,9 @@ def __init__(self, kernel_size: int = 11, kernel_sigma: float = 1.5, k1: float =
         # Cash kernel between calls.
         self.kernel = _fspecial_gauss_1d(kernel_size, kernel_sigma)
 
-    def forward(self, prediction: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
+    def forward(self,
```

Comment:
Should there be the similar approach to interface for `_Loss` as it was introduced for `BaseFeatureMetric`? the same applies for other children of `_Loss`.

Reply:
I consider this extra level of abstraction to be redundant